### DISCUSSION TITLE
Is there a way to get the explanations and images for later reference ?

### DISCUSSION THREAD

**Discussion by LuGodi** [pcottle/learnGitBranching#1201] [Q&A] [answered]
This may be a silly question but I love the explanations and examples, I would like to know if theres a markdown or text file with the whole explanations without having to go through the levels and clicks so I can reference it later ? 

Amazing job ! I enjoyed every single minute of it.
Thank you :)

**Comment by pcottle** [comment]
Ah this is an interesting idea! One could do an export of all the raw text easily, but it wouldn't be very good without the visual git demonstrations.

It'd have to include those as well which makes this a bit of a harder problem. Not impossible though!

**Reply by LuGodi** [reply]
> Ah this is an interesting idea! One could do an export of all the raw text easily, but it wouldn't be very good without the visual git demonstrations.
> 
> It'd have to include those as well which makes this a bit of a harder problem. Not impossible though!

Thank you for answering!
Ah I have already seen the examples so it would be only for reminding myself of specific parts.
How can I get the raw text? I figured I would need a script to iterate through the files of each lesson and get the english text through regex or something.
Is there an  easier way ? Sorry for the stupid question I am still kind of learning ^^’

**Reply by pcottle** [reply]
Oh just the text is much easier. Yeah there's definitely a better way than trying to use regrexes... I'll try to generate something later this week. Have a lot of work on my plate at my actual job but once I get some time, this should be straightforward

**Comment by pcottle** [comment, marked as answer]
Here you go!

https://learngitbranching.js.org/generatedDocs/levels.html

**Reply by LuGodi** [reply]
Omg thank you so so so much ><

**Comment by pcottle** [comment]
Did a really basic version of this in:
https://github.com/pcottle/learnGitBranching/commit/4cfaa0ae021b7a62b136f9955aa407c11e14dc2c

It's a little rough without the images but maybe we can capture those manually one day and then inline them into the docs...
